### DISCUSSION TITLE
Pdo is not suported ?

### DISCUSSION THREAD

**Discussion by zhangzijun** [perftools/xhgui#423] [Q&A] [answered]
PdoSearcher 

```php
public function getForUrl($url, $options, $conditions = []): void
{
    throw NotImplementedException::notImplementedPdo(__METHOD__);
}
```

How can I use PdoSearcher ?

**Comment by glensc** [comment, marked as answer]
until someone implements it, you can not use a feature that is not implemented!

The features missing are tracked in the pinned issue:
- https://github.com/perftools/xhgui/issues/320
